go.mod

```diff
@@ -14,8 +14,8 @@ require (
 	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
 	github.com/ryanuber/go-glob v1.0.0 // indirect
 	github.com/sirupsen/logrus v1.4.2
-	github.com/smartystreets/goconvey v1.6.4
-	github.com/stretchr/testify v1.3.0 // indirect
+	go.opentelemetry.io/otel v0.20.0
```

Comment:
In other projects we are using v1.0.0-RC1 -> https://github.com/magneticio/vamp-cloud-lamia/blob/master/go.mod#L37-L41